test_robot_launch_transmission_test is flaky

```

2026-04-02T04:18:40.9675996Z     NO TESTS RAN
2026-04-02T04:18:40.9676202Z     -- run_test.py: return code 1
2026-04-02T04:18:43.1167853Z     -- run_test.py: verify result file '/home/runner/work/ros2_control_ci/ros2_control_ci/.work/target_ws/build/mujoco_ros2_control_tests/test_results/mujoco_ros2_control_tests/test_robot_launch_transmission_test.py.xunit.xml'
2026-04-02T04:18:43.1169128Z   >>>
2026-04-02T04:18:43.1169804Z build/mujoco_ros2_control_tests/test_results/mujoco_ros2_control_tests/test_robot_launch_transmission_test.py.xunit.xml: 12 tests, 0 errors, 1 failure, 2 skipped
2026-04-02T04:18:43.1170663Z - robot_launch_test.TestFixture test_set_pause_and_step_simulation
2026-04-02T04:18:43.1171064Z   <<< failure message
2026-04-02T04:18:43.1171299Z     Traceback (most recent call last):
2026-04-02T04:18:43.1172194Z       File "/home/runner/work/ros2_control_ci/ros2_control_ci/.work/target_ws/src/ros-controls/mujoco_ros2_control/mujoco_ros2_control_tests/test/robot_launch_test.py", line 537, in test_set_pause_and_step_simulation
2026-04-02T04:18:43.1173607Z         self.assertAlmostEqual(paused_clock_time, clock_time, delta=MUJOCO_TIMESTEP)
2026-04-02T04:18:43.1174258Z     AssertionError: 12.732 != 12.734 within 0.002 delta (0.002000000000000668 difference)
2026-04-02T04:18:43.1174846Z   >>>
```

https://github.com/ros-controls/ros2_control_ci/issues/724
https://github.com/ros-controls/ros2_control_ci/issues/725

--- a/mujoco_ros2_control_tests/test/robot_launch_test.py
+++ b/mujoco_ros2_control_tests/test/robot_launch_test.py
@@ -534,7 +534,9 @@ class TestFixture(unittest.TestCase):
         self.spin_until(clock_settled, timeout=2.0)
         paused_clock_time = clock_time
         self.spin_until(lambda: False, timeout=1.5)
-        self.assertAlmostEqual(paused_clock_time, clock_time, delta=MUJOCO_TIMESTEP)
+        # Allow a small epsilon beyond MUJOCO_TIMESTEP to absorb floating-point
+        # rounding that accumulates when converting sec + nanosec * 1e-7.
+        self.assertAlmostEqual(paused_clock_time, clock_time, delta=MUJOCO_TIMESTEP + 1e-7)
 
         # Setting the simulation to paused again should still succeed
         future = pause_client.call_async(pause_req)

